ui/user/src/lib/components/Obot.svelte

```diff
@@ -230,7 +230,7 @@
 							/>
 						{/key}
 					{:else if layout.sidebarConfig}
-						<SidebarConfig bind:project bind:currentThreadID {assistant} />
+						<SidebarConfig bind:project {assistant} />
```

Comment:
I think this was dropped by accident. I'll double-check.

Reply:
Didn't affect anything AFAIK. I'm going to drop this change to avoid breaking anything unintentionally.